libs/saksbehandling-common/src/main/kotlin/event/SoeknadInnsendt.kt

```diff
@@ -10,8 +10,14 @@ interface ISoeknadInnsendt {
     val adressebeskyttelseKey get() = "@adressebeskyttelse"
     val fnrSoekerKey get() = "@fnr_soeker"
     val dokarkivReturKey get() = "@dokarkivRetur"
-    val eventNameInnsendt get() = "soeknad_innsendt"
-    val eventNameBehandlingBehov get() = "trenger_behandling"
 }
 
 object SoeknadInnsendt : ISoeknadInnsendt
+
+enum class SoeknadInnsendtHendelseType(val eventname: String) : EventnameHendelseType {
```

Comment:
Denne også, eit eller anna lib

Reply:
Blir ikke det litt micromanagement å kun ha et lite enum alene hvis alle som bruker det allerede har modulen den liger i?

Reply:
Jo, vi treng ikkje ein eigen modul eller noko for denne, men ideelt sett synes eg denne hadde passa i libs/etterlatte-soeknad - som vi jo ikkje har. Vi kan jo eventuelt flytte denne dit viss vi lagar noko sånt ein gong.